mixins/localize-dynamic-mixin.js

```diff
@@ -0,0 +1,41 @@
+import { getLocalizeOverrideResources } from '../helpers/getLocalizeResources.js';
+import { LocalizeMixin } from './localize-mixin.js';
+
+const defaultLangs = ['ar', 'cy-gb', 'cy', 'da', 'de', 'en', 'es-es', 'es', 'fr-ca', 'fr', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-tw', 'zh'];
```

Comment:
I love that all the default languages are just handled for folks, but I'm worried about what happens during the period of time from when we add a new language here (e.g. `cy` was added recently) and when downstream components actually add translations for that language.

I think what would happen is it would call `await importFunc('cy')` which would 404 and may blow up on the next line? The two alternatives I can think of:

1. We don't have a default list like this -- each implementation **must** provide the list of languages they support. We could maybe have `en` as the default since everyone should at least support that.
2. We somehow handle the 404 by detecting it and moving on to the next language in the list. I don't love having a 404 be a non-error though as users will see it in their consoles and our devs may not notice and never fix the underlying issue.

Reply:
Yeah, I was hoping we could just be conservative on adding new languages here. Maybe we wait for serge to send out PRs for the new language and then give people `x` amount of time before merging it in here. Perhaps it's a breaking change each time 😬, though it shouldn't be super frequent. Unless we can be smart and add error handling as a backup as well. With telemetry even?

Otherwise, forcing each component to declare supported languages isn't the end of the world.